packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -1260,9 +1263,11 @@ export class UmbrellaStore<M extends BaseMetadata> {
             return result;
           }
 
+          const subscriptions = threadSubscriptions.get().subscriptions;
+
           const threads = this.outputs.threads
             .get()
-            .findMany(roomId, query ?? {}, "asc");
+            .findMany(roomId, query ?? {}, "asc", subscriptions);
```

Comment:
Do you know if this will update/re-render if the subscriptions change but not the threads (e.g. when unsubscribing from a thread)? I can't fully remember how "threadifications" and subscriptions signals derive/link themselves

Reply:
I noticed a `threadSubscriptions.get()` call in this callback, and since `threadSubscriptions` is a signal too, the derived signal will recompute whenever `threadSubscriptions` changes.